alembic/versions/373ef4569b93_correlations_table.py

```diff
@@ -0,0 +1,71 @@
+"""correlations table
+
+Revision ID: 373ef4569b93
+Revises: 335c2bfd99a6
+Create Date: 2016-04-25 14:11:28.373859
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+from socorrolib.lib import jsontype
+
+# revision identifiers, used by Alembic.
+revision = '373ef4569b93'
+down_revision = '335c2bfd99a6'
+
+
+def upgrade():
+    op.create_table(
```

Comment:
I'm not trolling but I am stirring. 
Django can talk SQL to any PG table without necessarily have created its table. 

Adrian's point is that it's mixing the streams into one giant pot. I.e. we're using Django for too much where as things like socorro.external.postgres.* is all about data and the stuff that django does is all about presentation. 
Also, I'm such a Django-nut that I wonder if sometimes I see Django as the hammer that hammers any kind of nail. 

But think about it. You get a really powerful ORM query language. You can generate forms from the models. You can use django migrations. 

I'd love to take this, as an idea, seriously again late in 2016 or some time next year.